src/components/home/AssetList.tsx

```diff
@@ -41,25 +43,36 @@ export default function AssetList() {
           alignItems="center"
           justifyContent="space-between"
           borderBottomWidth={index === positions.length - 1 ? 0 : 1}
+          borderTopWidth={index === 0 ? 1 : 0}
           borderColor="$borderNeutralSoft"
         >
           <View flexDirection="row" alignItems="center" paddingVertical={vs(10)} flex={1}>
             <View flexDirection="row" gap={ms(10)} flex={1} alignItems="center">
-              <AssetLogo uri={assetLogos[symbol as keyof typeof assetLogos]} width={ms(32)} height={ms(32)} />
+              <AssetLogo uri={assetLogos[symbol as keyof typeof assetLogos]} width={ms(40)} height={ms(40)} />
               <View gap={ms(5)}>
                 <Text fontSize={ms(15)} fontWeight="bold">
                   {symbol}
                 </Text>
-                <Text>
-                  {(
-                    Number(rates.find((r: { market: string; rate: bigint }) => r.market === market)?.rate) / 1e18
-                  ).toLocaleString(undefined, {
-                    style: "percent",
-                    minimumFractionDigits: 2,
-                    maximumFractionDigits: 2,
-                  })}{" "}
-                  APR
+              </View>
+            </View>
+            <View flexDirection="row" gap={ms(10)} alignItems="center" justifyContent="flex-end">
```

Comment:
for `gap` values it is preferred to use tamagui's notation. you can check which tamagui sizes match with the required gap in `tamagui.config` file